docs/01-introduction/legal-info.md

```diff
@@ -0,0 +1,83 @@
+<!---
+ * Copyright © 2021 - 2025 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+# Storing Legal Information
+
+Each file representation in the project data may have legal metadata associated with it.
+This metadata is stored in the [FileValue](../02-dsp-ontologies/knora-base.md#filevalue) that represents the file.
+
+!!! warning "Legal Information Will Become Mandatory"
+
+    Currently, the legal information on the FileValue is optional. This will change in the future when License, Authorship, and Copyright Holder will become mandatory.
```

Comment:
This info could be added to the deprecations page, because - while technically not a deprecation, just an API change - eventually it will no longer be possible to create assets without legal info. So I think it would fit quite well with the other deprecations.